apps/wallet-mobile/src/features/Receive/useCases/DescribeSelectedAddressScreen.tsx

```diff
@@ -31,6 +42,13 @@ export const DescribeSelectedAddressScreen = () => {
     copy(selectedAddress)
   }
 
+  React.useEffect(() => {
+    isSingle &&
+      multipleAddressesUsed &&
+      isShowingMultipleAddressInfo &&
+      openModal(strings.singleOrMultiple, <Modal />, modalHeight)
+  }, [isShowingMultipleAddressInfo, isSingle, multipleAddressesUsed, openModal, strings.singleOrMultiple])
```

Comment:
nitpicky consistency no worries, I'm considering that this should be part of the useAddressStatus -> status should report this things on top of my head.
```suggestion
  }, [isShowingMultipleAddressInfo, isSingle, isMultipleAddressesUsed, openModal, strings.singleOrMultiple])
```